.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
         "no-useless-escape": "off",
         "no-inner-declarations": "off",
         "no-constant-condition": ["error", { "checkLoops": false }],
+        "no-restricted-imports": ["error", { "paths": ["vscode-azureappservice/out/src/extensionVariables", "**/*/extension.bundle"] }],
```

Comment:
So for `**/*/extension.bundle` I agree we want that for `src`, but we actually want the opposite for `test` (people _should_ use "extension.bundle" and _not_ `src` because we want tests running against the webpacked code on the build machines)

It looks like this repo doesn't have a test-specific "eslintrc.js" yet, so I'm curious why the build passes when we have extension.bundle imports here: https://github.com/microsoft/vscode-azurestaticwebapps/blob/main/test/selectBuildConfigs.test.ts#L9

Reply:
Oops. I need to use "patterns" instead of "paths" to use globs - so the build passed because the `**/*extension.bundle` rule wasn't actually working.

I'll create a test specific .eslintrc to prevent the error appearing in tests.